snet_cli/commands.py

```diff
@@ -504,3 +506,212 @@ def transact(self):
             return receipt, events
         else:
             self._error("Cancelled")
+
+
+class ServiceCommand(ClientCommand):
+
+    def _tar_imports(self, paths):
+        max_depth = 0
+        deepest_path = PurePath()
```

Comment:
Adding description.
I'm sorting files in the .tar in a predictable way because otherwise a different tar with a different checksum will be produced and you could have a tar with the same files in it with a different ipfs hash. I'm abiding by GNU tar's default sorting order.